bots/csr/src/main/java/org/openjdk/skara/bots/csr/PullRequestWorkItem.java

```diff
@@ -49,8 +49,8 @@
  */
 class PullRequestWorkItem implements WorkItem {
     private final static String CSR_LABEL = "csr";
-    private final static String CSR_UPDATE_MARKER = "<!-- csr: 'update' -->";
-    private static final String PROGRESS_MARKER = "<!-- Anything below this marker will be automatically updated, please do not edit manually! -->";
+    protected final static String CSR_UPDATE_MARKER = "<!-- csr: 'update' -->";
+    protected static final String PROGRESS_MARKER = "<!-- Anything below this marker will be automatically updated, please do not edit manually! -->";
```

Comment:
Does this work? I thought you needed to use the default package protected (by removing any `private` or `protected` prefix).

Reply:
Yes, this works. Ok, I will remove the prefix.